client/src/app/site/agenda/components/agenda-list/agenda-list.component.html

```diff
@@ -13,6 +13,7 @@
     </div>
 
 </os-head-bar>
+<mat-drawer-container class="on-transition-fade">
```

Comment:
could fail in travis, try to re-align that

Reply:
so apparently prettify-write does not all of these things